server/src/main/java/org/opensearch/gateway/remote/ClusterMetadataManifest.java

```diff
@@ -751,6 +1033,9 @@ public Builder(ClusterMetadataManifest manifest) {
             this.clusterUUIDCommitted = manifest.clusterUUIDCommitted;
             this.routingTableVersion = manifest.routingTableVersion;
             this.indicesRouting = new ArrayList<>(manifest.indicesRouting);
+            this.diffManifest = manifest.diffManifest;
```

Comment:
Do we need to add clusterblocks, discoveryNodes, metadata version, transient settings here ?

Reply:
Added